src/main/java/org/folio/bulkops/config/properties/RemoteFileSystemProperties.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.bulkops.config.properties;
```

Comment:
We don't need this configuration as it is already incapsulated in the folio-s3-client library.